internal/compiler/tests/syntax/new_syntax/two_way_input_output.slint

```diff
@@ -304,6 +305,9 @@ export component C12 {
     in property <bool> in5 <=> inout1;
 //                         ^error{Cannot link input property}
 
+    in property <bool> boggus1 <=> btn.internal;
+//                                     ^error{The property 'internal' is private. Annotate it with}
```

Comment:
I was wondering if there's something missing here, but I guess it's fine as this is just matching the message, which will be longer than that.

Reply:
yes, i was just lazy to type the full message.